compiler/src/dotty/tools/dotc/printing/RefinedPrinter.scala

```diff
@@ -320,7 +324,8 @@ class RefinedPrinter(_ctx: Context) extends PlainPrinter(_ctx) {
         if (name.isTypeName) typeText(txt)
         else txt
       case tree @ Select(qual, name) =>
-        if (tree.hasType && tree.symbol == defn.QuotedExpr_~ || tree.symbol == defn.QuotedType_~) keywordStr("~(") ~ toTextLocal(qual) ~ keywordStr(")")
+        if (tree.hasType && tree.symbol == defn.QuotedExpr_~) keywordStr("~(") ~ toTextLocal(qual) ~ keywordStr(")")
+        else if (tree.hasType && tree.symbol == defn.QuotedType_~) typeText("~(") ~ toTextLocal(qual) ~ typeText(")")
```

Comment:
Why typeText instead if keywordStr? (I don't know enough about the syntax highlighting stuff to be able to tell).

Reply:
It looks like this:
<img width="514" alt="screenshot 2019-02-12 at 10 33 08" src="https://user-images.githubusercontent.com/3648029/52625590-9e36ee80-2eb1-11e9-9533-26aabd38f3f0.png">

I added it because in some large examples it was easyer to see which where types splices and which  where terms splices.